src/components/graph/menus/editable-title.js

```diff
@@ -30,28 +30,27 @@ const useStyles = makeStyles((theme) => ({
         ),
         display: 'flex',
         alignItems: 'center',
-    },
+    }),
     nodeNameTitle: {
         flexGrow: 1,
         fontWeight: 'bold',
     },
-}));
+};
 
 export const EditableTitle = ({ name, onClose, onChange }) => {
     const [openEditTitle, setOpenEditTitle] = useState(false);
-    const classes = useStyles();
     const intl = useIntl();
 
     return (
-        <div className={classes.header}>
+        <Box sx={styles.header}>
             <IconButton
                 size={'small'}
                 onClick={() => setOpenEditTitle(true)}
                 disabled={onChange === undefined}
             >
                 <EditIcon />
             </IconButton>
-            <OverflowableText text={name} className={classes.nodeNameTitle} />
+            <OverflowableText text={name} sx={styles.nodeNameTitle} />
```

Comment:
I'm not sure sx will work here
OverflowableText is defined this way :
```
                <Box
                    {...props}
                    ref={element}
                    children={children || text}
                    className={className}
                    sx={overflowStyle.overflow}
                />
```
Since sx is defined again, it might be overridden ?

Reply:
After further testing, it works